app/uk/gov/hmrc/merchandiseinbaggage/controllers/CannotUseServiceController.scala

```diff
@@ -31,10 +31,18 @@ class CannotUseServiceController @Inject()(
 
   private val backUrl = Call("GET", "#")
 
+  private def backButtonUrl(implicit request: DeclarationJourneyRequest[_]): Call = {
```

Comment:
i guess will improve this when we will introduce back button in the navigator

Reply:
👍